MonoGame.Framework/Utilities/PlatformInfo.cs

```diff
@@ -38,6 +38,8 @@ public static MonoGamePlatform MonoGamePlatform
                 return MonoGamePlatform.PlayStation4;
 #elif PSVITA
                 return MonoGamePlatform.PSVita;
+#elif STADIA
+                return MonoGamePlatform.Stadia;
```

Comment:
@Jjagg @harry-cpp - May be good to clean up these #ifs and replace it with partial class stuff?

Reply:
This is one of those rare moments where I think #if is what we actually want.

Reply:
Yeah, we chose to do it like this. Code is fairly recent.